include/swift/Basic/LangOptions.h

```diff
@@ -41,13 +41,15 @@ namespace swift {
     OS,
     /// The active arch target (x86_64, i386, arm, arm64, etc.)
     Arch,
-    /// The active endianness target (big or little)
+    /// The active arch target endianness (big or little)
     Endianness,
+    /// The active arch target pointer bit width (_32 or _64)
+    PointerBitWidth,
```

Comment:
Cf. `__POINTER_WIDTH__` in Clang, `getArchPointerBitWidth` in LLVM,  `target_pointer_width` in Rust.